java/src/jmri/jmrix/lenz/hornbyelite/messageformatters/EliteMessageFormattersBundle_nl.properties

```diff
@@ -0,0 +1,11 @@
+# XNetMessageFormatterBundle_es.properties
+#
+# Message translation strings for the XpressNet Monitor from XNetMessages and XNetReplies
+#
+# Dutch properties for the jmri.jmrix.lenz.messageformatters Package
+# Translation by Egbert Broerse 2017
+
+
+# XNetReply Loco information
+XNetReplyLocoEliteSLabel                    = Elite Snelheid-/Rijrichting-information: Loc
+XNetReplyLocoEliteFLabel                    = Elite Functie-informatie: Loc
```

Comment:
This file doesn't have an empty line at the end of the file.

Reply:
fixed